src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsService.java

```diff
@@ -1,15 +1,32 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.services.reports;
 
-import org.apache.commons.lang3.NotImplementedException;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.reports.EnvelopeCountSummaryRepository;
 
 import java.time.LocalDate;
 import java.util.List;
 
+import static java.util.stream.Collectors.toList;
+
 @Service
 public class ReportsService {
 
+    private final EnvelopeCountSummaryRepository repo;
+
+    public ReportsService(EnvelopeCountSummaryRepository repo) {
+        this.repo = repo;
+    }
+
     public List<EnvelopeCountSummary> getCountFor(LocalDate date) {
-        throw new NotImplementedException("Not yet implemented");
+        return repo
+            .getReportFor(date)
+            .stream()
```

Comment:
i would favour for repository returning stream instead. especially when it has such possibility and reports make sense to be streamed instead of evaluated as early as possible

not part of the style though

Reply:
I'll do that on next PR (to not mess with repo layer and its' tests in this PR)